File: C programs/C++/SimpleInterest.cpp
#include<iostream>
using namespace std;        // A c++ program to find the simple interest, system("cls") is used to clear 
int main(){                 // the screen only you may use or may not
    int principal, rate, time, simple_interest;
    system("cls");
    cout<<"Enter principal amount\n";
    cin>>principal;
    system("cls");
    cout<<"Enter rate of interest\n";
    cin>>rate;
    system("cls");
    cout<<"Enter the time period\n";
    cin>>time;
    system("cls");
    simple_interest=(principal*rate*time)/100;
    cout<<"Simple Interest is = "<<simple_interest;
    return 0;
}

File: C programs/Java_Projects/SimpleInterest.java
import java.util.Scanner;

public class SimpleInterest {
    public static void main(String[] args) {
        int principal, rate, time,si;
        System.out.println("Enter the principal rate and time");
        Scanner s=new Scanner(System.in);
        principal=s.nextInt();
        rate=s.nextInt();
        time=s.nextInt();
        si=(principal*rate*time)/100;
        System.out.println("Simple Interest is = "+si);
    }
}
